### PR TITLE
Add annotation about python3

### PR DESCRIPTION
**Pull request by suanmeiguo**
This addition is in regarding to #2062: add explanation about python3 for RTD installation.

### CODE REVIEW

File: docs/install.rst [hunk 1]
```diff
@@ -9,7 +9,8 @@ It will get you to a point of having a local running instance.
 First, obtain `Python 2.7`_ and virtualenv_ if you do not already have them. Using a
 virtual environment will make the installation easier, and will help to avoid
 clutter in your system-wide libraries. You will also need Git_ in order to
-clone the repository.
+clone the repository. If you plan to import python3 project to your RTD then you'll
+need to install python3 with virtualenv in your system as well.
```
Comment (agjohnson) [inline docs/install.rst]: 'python3' -> 'Python 3'

**Comment by agjohnson** [comment]
Thanks for the addition! I noted a copy issue, but looks good otherwise

**Comment by suanmeiguo** [comment]
Awesome. Good catch. Just updated.

**Comment by agjohnson** [comment]
Giddyup. Thanks!

**Pull request merged**